addon/components/cfb-form-editor/question/default.js

```diff
@@ -0,0 +1,89 @@
+import { getOwner } from "@ember/application";
+import { action, computed } from "@ember/object";
+import { inject as service } from "@ember/service";
+import RenderComponent from "ember-validated-form/components/validated-input/-themes/uikit/render";
+
+import { TYPE_MAP } from "ember-caluma/lib/field";
+import layout from "ember-caluma/templates/components/cfb-form-editor/question/default";
+
+export default class CfbFormEditorQuestionDefault extends RenderComponent {
+  layout = layout;
+
+  @service router;
+
+  @computed("model.{id,__typename}")
+  get question() {
+    const raw = {
+      ...this.model.get("data"),
+      ...this.model.get("change"),
+      isRequired: "false",
```

Comment:
This needs a comment

Reply:
Done. I also added a comment in `this.fields` about the structure of `this.value`.